packages/devtools_app/lib/src/shared/preferences.dart

```diff
@@ -164,7 +164,7 @@ class InspectorPreferencesController extends DisposableController
             };
 
             // The isolate is still paused, listen for when it becomes unpaused.
-            debuggerState?.isPaused.addListener(pausedListener);
+            addAutoDisposeListener(debuggerState?.isPaused, pausedListener);
```

Comment:
just want to the disposal of this listener doesn't cause any issues when the disposal code path is hit after the listener has already been removed?

Reply:
The removeListener code only removes if it can find the listener, so I think this should be fine.
I'm also tested this manually and no errors came up :)